dotnet/src/SemanticKernel.UnitTests/Utilities/InternalTypeConverterTests.cs

```diff
@@ -0,0 +1,142 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.ComponentModel;
+using System.Globalization;
+using Xunit;
+
+namespace SemanticKernel.UnitTests.Utilities;
+public class InternalTypeConverterTests
```

Comment:
Small spacing here :)
```suggestion

public class InternalTypeConverterTests
```